apps/to-do-api/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "to-do-api",
+  "version": "0.0.0",
+  "description": "Azure Function REST API for To Do List",
+  "license": "MIT",
+  "main": "./dist/main.js",
+  "scripts": {
+    "clean": "shx rm -rf ./dist ./src/generated",
+    "build": "npm run generate && tsc",
```

Comment:
- `generate` is needed at every run?
- if you want, you can use `esbuild` instead of `tsc` (like https://github.com/pagopa/io-fims/blob/main/apps/op-app/package.json#L35)

Reply:
nope, you are right.

if there must be a dependency, then I think we should handle it with turbo tasks.